UNRELEASED.md

```diff
@@ -16,6 +16,8 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 
 ### Development workflow
 
+- Made it possible to prevent [“yarn splash”](https://github.com/Shopify/polaris-react/tree/master/scripts/splash) from automatically running alongside `yarn dev`: `yarn dev-no-splash` or `DISABLE_SPLASH=1 yarn dev` ([#2372](https://github.com/Shopify/polaris-react/pull/2372))
```

Comment:
`yarn dev-no-splash` is not an option is it?

Reply:
Apart from that everything looks good. 🎩 as well.

Reply:
Thank you, good catch!